.scalafmt.conf

```diff
@@ -3,6 +3,7 @@ version = 3.8.2
 align.preset = more
 runner.dialect = scala212
 maxColumn = 150
+lineEndings=preserve
```

Comment:
I don't know if it's a good idea to mix UNIX and Windows line endings. Usually IDEs don't have a problem working with UNIX line ends ona a Windows PC.